packages/core/src/DreifussWysiwygEditor.tsx

```diff
@@ -154,14 +147,13 @@ export default function DreifussWysiwygEditor(props: EditorProps) {
             <ToolbarLink />
           </Modal>
           <Divider type={DividerType.vertical} />
-          <ToolbarImage icon={<ImageIcon />} />
```

Comment:
removed this as we have it in line 148 which will be uncommented in the image plugin PR

Reply:
👍🏾